contracts/AelinPool.sol

```diff
@@ -236,11 +231,7 @@ contract AelinPool is AelinERC20, MinimalProxyFactory, IAelinPool {
             NftCollectionRules memory nftCollectionRules = nftCollectionDetails[collectionAddress];
             require(nftCollectionRules.collectionAddress == collectionAddress, "collection not in the pool");
 
-            if (nftCollectionRules.purchaseAmountPerToken) {
```

Comment:
this is not right. we want to keep the following logic with a minor change:

if (nftCollectionRules.purchaseAmount > 0) {
    maxPurchaseTokenAmount += nftCollectionRules.purchaseAmount * tokenIds.length;
}

and we want to remove this entire if statement. we are no longer using the nftWalletUsedForPurchase mapping anymore:

            if (!nftCollectionRules.purchaseAmountPerToken && nftCollectionRules.purchaseAmount > 0) {
                require(!nftWalletUsedForPurchase[collectionAddress][msg.sender], "wallet already used for nft set");
                nftWalletUsedForPurchase[collectionAddress][msg.sender] = true;
                maxPurchaseTokenAmount += nftCollectionRules.purchaseAmount;
            }

Reply:
Oh yes, sorry my mistake